Makefile

```diff
@@ -176,7 +176,7 @@ endif
 
 .PHONY: images
 images:
-	$(ENGINE) build -f Dockerfile.rhel8 -t "$(IMAGE):$(VERSION)" -t "$(IMAGE):$(MUTABLE_TAG)" ./
+	$(ENGINE) build -f Dockerfile.ocp -t "$(IMAGE):$(VERSION)" -t "$(IMAGE):$(MUTABLE_TAG)" ./
```

Comment:
Huh? Seems unrelated?

Reply:
Sorry, i'll remove this.  I'll put it in a new PR.  Someone changed the name of the dockerfile, but never updated the make file which I use to do my builds.